src/containers/ImageModal.js

```diff
@@ -13,6 +13,12 @@ const leftOffset = () => {
 export default class ImageModal extends React.Component {
   static propTypes = {url: string, onCancel: func}
 
+  handleClick (event) {
+    const { onCancel } = this.props
+    event.persist()
```

Comment:
it had to do with react synthetic pooling, but I think that may actually have only been an issue doing console.logging
https://facebook.github.io/react/docs/events.html